requirements-development.txt

```diff
@@ -1,4 +1,4 @@
--r production.txt
+-r requirements.txt
```

Comment:
This file was refactored in current [develop](https://github.com/turicas/rows/tree/develop) branch. You need to rebase your branch based on this new [develop](https://github.com/turicas/rows/tree/develop).